site2/docs/client-libraries-node.md

```diff
@@ -284,7 +284,7 @@ Pulsar Node.js readers have the following methods available:
 | :----- | :---------- | :---------- |
 | `readNext()` | Receives the next message on the topic (analogous to the `receive` method for [consumers](#consumer-operations)). When the message is available, the Promise object run executor function and get message object. | `Promise<Object>` |
 | `readNext(Number)` | Receives a single message from the topic with specific timeout in milliseconds. | `Promise<Object>` |
-| `hasNext()` | Return whether Proker has next message in target topic. | `Boolean` |
+| `hasNext()` | Return whether Broker has next message in target topic. | `Boolean` |
```

Comment:
In the rest of the document I saw "the broker" or "the Pulsar broker". I think it makes sense to stay consistent through the whole documentation.

Reply:
Thanks for the feedback. I've made the change below.